docs/git/getting-started.md

```diff
@@ -0,0 +1,176 @@
+# Git Getting Started
+
+The following is a getting started guide for using [git] while developing
+Cardano SL.
+
+Its probably best to avoid the use of GUIs that try to make the use of [git]
+easier. Git is a complex tool with many sub commands and each sub command has
+many options. This makes it difficult for a GUI to provide the required
+flexibility. Its also probably best to avoid the use of the [gitflow]
+extension as it provides zero advantage over a solid intermediate level
+understanding of git itself.
+
+Finally git is already relatively customizable and with the addition of custom
+shell functions, shell aliases and shell scripts customization is almost
+limitless.
+
+In the following, the most standard versions of each command will be used and
+when the command uses options, the long option name will be used. It should be
+noted however that if the `${HOME}/.gitconf` below makes the command
```

Comment:
here it should also be `${HOME}/.gitconfig`

Reply:
Fixed.